.editorconfig

```diff
@@ -0,0 +1,17 @@
+# editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
```

Comment:
Ugh, that was initially auto-generated, that's why it has 2. I hate this type of indentation, I find it really confusing. I only used `tab-size: 2` in the homepage of Bliss because there was no space and I was trying to prevent scrolling (and I know that many people like a size of 2, I'm in the minority there).